NEWS.md

```diff
@@ -42,6 +42,14 @@
   (#174). Previously it would always refer to the environment of the
   outermost quosure.
 
+* The new functions `cnd_warn()` and `cnd_inform()` transform
+  conditions to warnings or messages before signalling them.
+
+* `signal_deprecated()` and `warn_obsolete()` depreciate functions. A
```

Comment:
I'm slight concerned that we're creating a parallel set of terminology to deprecated and defunct in base R.